*** cub-1.12.0/cmake/CubCudaConfig.cmake-dist	2021-05-04 13:47:50.794755948 +0900
--- cub-1.12.0/cmake/CubCudaConfig.cmake	2021-05-04 13:48:05.265008298 +0900
***************
*** 4,10 ****
  # Architecture options:
  #
  
! set(all_archs 35 37 50 52 53 60 61 62 70 72 75 80)
  set(arch_message "CUB: Explicitly enabled compute architectures:")
  
  # Thrust sets up the architecture flags in CMAKE_CUDA_FLAGS already. Just
--- 4,10 ----
  # Architecture options:
  #
  
! set(all_archs 86)
  set(arch_message "CUB: Explicitly enabled compute architectures:")
  
  # Thrust sets up the architecture flags in CMAKE_CUDA_FLAGS already. Just
