*** cupy-9.2.0/cupy_setup_build.py-dist	2021-07-14 01:48:20.247856582 +0900
--- cupy-9.2.0/cupy_setup_build.py	2021-07-14 01:49:23.316036475 +0900
***************
*** 18,23 ****
--- 18,29 ----
  from install.build import PLATFORM_LINUX
  from install.build import PLATFORM_WIN32
  
+ try:
+     # This is to avoid getting numpy imported inside other modules and
+     # overwritting setuptools compilers (#5476)
+     import numpy.distutils  # NOQA
+ except Exception:
+     pass
  
  # Cython requirements (minimum version and versions known to be broken).
  # Note: this must be in sync with setup_requires defined in setup.py.
